content/en/docs/tasks/inject-data-application/distribute-credentials-secure.md

```diff
@@ -364,72 +377,74 @@ another Pod which consumes a secret with test environment credentials.
+   For example, if your actual password is `S!B\*d$zDsb=`, you should execute the command as follows:
 
-    ```shell
-    kubectl create secret generic dev-db-secret --from-literal=username=devuser --from-literal=password='S!B\*d$zDsb='
-    ```
+   ```shell
+   kubectl create secret generic dev-db-secret --from-literal=username=devuser --from-literal=password='S!B\*d$zDsb='
+   ```
 
-    You do not need to escape special characters in passwords from files (`--from-file`).
-    {{< /note >}}
+   You do not need to escape special characters in passwords from files (`--from-file`).
+   {{< /note >}}
 
 1. Create the Pod manifests:
 
-    ```shell
-    cat <<EOF > pod.yaml
-    apiVersion: v1
-    kind: List
-    items:
-    - kind: Pod
-      apiVersion: v1
-      metadata:
-        name: prod-db-client-pod
-        labels:
-          name: prod-db-client
-      spec:
-        volumes:
-        - name: secret-volume
-          secret:
-            secretName: prod-db-secret
-        containers:
-        - name: db-client-container
-          image: myClientImage
-          volumeMounts:
-          - name: secret-volume
-            readOnly: true
-            mountPath: "/etc/secret-volume"
-    - kind: Pod
-      apiVersion: v1
-      metadata:
-        name: test-db-client-pod
-        labels:
-          name: test-db-client
-      spec:
-        volumes:
-        - name: secret-volume
-          secret:
-            secretName: test-db-secret
-        containers:
-        - name: db-client-container
-          image: myClientImage
-          volumeMounts:
-          - name: secret-volume
-            readOnly: true
-            mountPath: "/etc/secret-volume"
-    EOF
-    ```
-    Note how the specs for the two Pods differ only in one field; this facilitates creating Pods with different capabilities from a common Pod template.
+   ```shell
+   cat <<EOF > pod.yaml
+   apiVersion: v1
+   kind: List
+   items:
+   - kind: Pod
+     apiVersion: v1
+     metadata:
+       name: prod-db-client-pod
+       labels:
+         name: prod-db-client
+     spec:
+       volumes:
+       - name: secret-volume
+         secret:
+           secretName: prod-db-secret
+       containers:
+       - name: db-client-container
+         image: myClientImage
+         volumeMounts:
+         - name: secret-volume
+           readOnly: true
+           mountPath: "/etc/secret-volume"
+   - kind: Pod
+     apiVersion: v1
+     metadata:
+       name: test-db-client-pod
+       labels:
+         name: test-db-client
+     spec:
+       volumes:
+       - name: secret-volume
+         secret:
+           secretName: test-db-secret
+       containers:
+       - name: db-client-container
+         image: myClientImage
+         volumeMounts:
+         - name: secret-volume
+           readOnly: true
+           mountPath: "/etc/secret-volume"
+   EOF
+   ```
+
+   Note how the specs for the two Pods differ only in one field; this facilitates creating Pods
```

Comment:
@windsonsea , can we add this under a Note section ({{< note >}} and {{< /note >}}) like below so that it will more highlighted.
```suggestion
   {{< note >}}
   How the specs for the two Pods differ only in one field; this facilitates creating Pods
   with different capabilities from a common Pod template.
   {{< /note >}}
```

Reply:
This is my suggestion, WDYT?

Reply:
Good idea. Implemented.

Reply:
Thanks @windsonsea for this quick update.